components/aws/sagemaker/tests/integration_tests/utils/kfp_client_utils.py

```diff
@@ -29,6 +30,13 @@ def wait_for_job_completion(client, run_id, timeout):
     return status
 
 
+def wait_for_job_to_start(client, run_id):
+    time.sleep(10)
```

Comment:
Is there any dynamic way to query the job rather than relying on a fixed interval?

Reply:
the sleep isn't really necessary here, also this method should be removed soon once we add the bot and check for completion.